src/views/GraphDetail.vue

```diff
@@ -387,7 +387,7 @@ export default Vue.extend({
       const token = getLoginToken();
 
       if (token !== null) {
-        return `&loginToken=${token}`;
+        return `loginToken=${token}`;
```

Comment:
I think you want to keep the `#` here; in case there is no token, line 96 above will have a `#` in the URL with nothing following it. Cleaner would be to keep the `#` here so that it won't appear in the URL at all if there's no token.

Reply:
I had moved it to the `url` param, above in this file, so that if we add another parameter to the fragment it would work as intended. Happy to make this change if you prefer

Reply:
Same concern that Roni raised. If we're concerned about supporting multiple fragments in the future, it'd probably be good to replace this computed function with a `fragment` computed function, which could aggregate any desired fragment sections, and return something like:

```
// Create this list somehow
const fragments = ['loginToken=abcd', ...];
return fragments.length ? `#${fragments.join("&")}` : '';
```

However, it's not something that really needs to be addressed in this PR, just a thought.

Reply:
Addressed in 5877967